FrontEndLib/Widget.h

```diff
@@ -372,7 +372,7 @@ class CWidget : public CAttachableObject
 	UINT        GetW() const {return this->w;}
 	UINT        GetH() const {return this->h;}
 
-	virtual void   Hide(const bool /*bPaint*/ = true) {this->bIsVisible = false;}
+	virtual void   Hide(const bool /*bPaint*/ = true);
```

Comment:
Remove commenting around bPaint here.

Reply:
Why was that commenting there in the first place? I've always been curious whenever I've encountered it. Was it just to stress that the default implementation does not use the argument?

Reply:
> Why was that commenting there in the first place? I've always been curious whenever I've encountered it. Was it just to stress that the default implementation does not use the argument?

@EvidentlyCube , great question.  This is actually to suppress a compiler warning.  Any time a method (virtual, in this case) has named arguments that aren't referenced in the body, the compiler may complain about an unused variable.  The way to suppress the variable is to not name the variable in the function signature that's alongside the definition.